### ISSUE TITLE
[Feature]: keyboard controls for caption delay

### ISSUE THREAD

**Issue by dimii27** [movie-web/movie-web#701]
### What feature do you want to add?

I think the title is self explanatory. If this starts a trend (with other keyboard shortcuts) it would be nice to see a settings page or guide in the readme explaining the keyboard shortcuts.

### Why do you want to have this feature?

context: I'm watching House MD and all the subtitles seem offset, and the delay I apply in the beginning of each episode is no longer fitting around the middle of the episode. It's not a big inconvenience but it surely is one.

Also, regarding caption delay, It would be nice to see a toggle for remembering the delay settings for the next episode.

### Anything other details to share?

To recap, I suggested 3 features:
- keyboard shortcuts for caption delay
- menu explaining all the keyboard shortcuts
- toggle to keep the caption delay setting for the next episode.

**Comment by Nithur-M** [comment]
I also faced this issue in some movies. Is there an option for caption delay?

**Comment by driftywinds** [comment]
I second this feature request



> I also faced this issue in some movies. Is there an option for caption delay?

Yes there is in the settings panel while a video is playing
